XamarinCommunityToolkit/Core/CameraMediaSource.shared.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Core
+{
+	// TODO: Check if we can do these changes in Xamarin.Forms to not have our own version of the MediaSource
+	// TODO 2: Is this even used?! Not for iOS it seems...
+	public class CameraMediaSource : MediaSource
```

Comment:
If/when `MediaElement` is dropped from Forms, we might also get the entire `MediaSource` in the Toolkit. Until that happens like this? Is this used in Android or UWP? It doesn't seem to be used in iOS.

Reply:
I think this is just to play the VideoRecord.

Reply:
It's used to retrieve the video information. Is used [here](https://github.com/xamarin/XamarinCommunityToolkit/pull/244/files#diff-61ca214876a712c1f7a458c977572116R10)

Reply:
Talked about this on stream, seems not to be used right now, but let's keep it for the Android and UWP versions. If its still not used then, let's remove it. #notetoself